google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java

```diff
@@ -100,7 +101,7 @@ public Bucket createBucketWithStorageClassAndLocation(String bucketName) {
     // [START createBucketWithStorageClassAndLocation]
     Bucket bucket = storage.create(BucketInfo.newBuilder(bucketName)
         // See here for possible values: http://g.co/cloud/storage/docs/storage-classes
-        .setStorageClass("COLDLINE")
+        .setStorageClass(StorageClass.COLDLINE)
```

Comment:
Yes, the integration tests pass. According to this url: https://cloud.google.com/storage/docs/managing-buckets#manage-class the class values are in upper case.